<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1591</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Menu">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
     <widget class="QLineEdit" name="dishDiscription">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>740</y>
        <width>451</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="dishName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>710</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>770</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dish Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>740</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dish discription</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>710</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dish name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addDish">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>860</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="menuTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>661</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Discription</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="sortMenuPrices">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>281</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sort by price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>10</y>
        <width>541</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dishPriceSpinBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>780</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sortByPopularity">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>10</y>
        <width>311</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sort by popularity</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openFile">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>690</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Open file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveFile">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>600</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Save file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="menuDishNameDelete">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>380</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>380</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DishName</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>410</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>330</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Orders</string>
     </attribute>
     <widget class="QPushButton" name="addOrder">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>540</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add Order</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>500</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> Dish name:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="menuBox">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>490</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="orderTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>661</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>591</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>591</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Dish name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Table number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dish amount</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>130</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Order number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>170</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Invoice</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>730</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Order number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="orderNumberChange">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>730</y>
        <width>271</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="changeButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>830</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>690</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Change:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>770</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Table number</string>
      </property>
     </widget>
     <widget class="QComboBox" name="newDishName">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>780</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>800</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>760</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dish name</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="amountOrderChange">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>800</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="tableNumberChange">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>770</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="orderNumberCheck">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>120</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>220</y>
        <width>561</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>-1</y>
         <width>561</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string> Invoice:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="checkLabel">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Order invoice</string>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>581</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>380</y>
        <width>561</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>2</y>
         <width>551</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="tableNumberLabel">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string> Table number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tableNumber"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>430</y>
        <width>561</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>561</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="dishAmountLabel">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string> Dish amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="dishAmount"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>600</y>
        <width>591</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelOrder">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>710</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel order</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
